tests/non-pytest/mypy-ignore-tests/flows_orderby.py

```diff
@@ -0,0 +1,29 @@
+# test the `orderby` param for `FlowsClient.list_flows()`
```

Comment:
What are non-pytest tests?

Is this a nose testsuite or something?

Reply:
We have a few tests which are checks that do not run as part of the normal testsuite. This is either for speed reasons or because they aren't written in pytest.
These are in a dedicated dir so that we can tell pytest not to recurse here.

`tox r -e test-lazy-imports` is a good example: those are _super_ slow to be running in a case like `tox run`, in which everything runs on each python version, but we still want/need them in CI.

In this case, the relevant run is `tox r -e mypy-test`, which runs `mypy` on the target files. But we don't want pytest to import/load these because :boom: